packages/kbn-optimizer/limits.yml

```diff
@@ -7,7 +7,7 @@ pageLoadAssetSize:
   banners: 17946
   bfetch: 22837
   canvas: 1066647
-  cases: 175000
+  cases: 180000
```

Comment:
This PR https://github.com/elastic/kibana/pull/157492 will reduce the limit substantially.

Reply:
Works for me.  Thanks for looking into it.